src/components/About/AvatarsGroup.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link } from "react-router-dom"
+import Avatar from "./Avatar"
+import useFetchContributorsAvatars from "../../hooks/useFetchContributorsAvatar";
+
+const TOTAL_AVATARS = 8
+
+type Props = {
+  repoUrl: string,
+  repoName: string,
+  repoTitle: string
+}
+
+const AvatarsGroup: React.FC<Props> = ({ repoUrl, repoName, repoTitle }) => {
+	const { contributors, total, loading, error } = useFetchContributorsAvatars(repoName);
+
+	// TODO: Add appropriate error message component, and a loading spinner
+	if (loading) return <div>Loading...</div>;
+	if (error) return <div>Error: {error.message}</div>;
```

Comment:
Just a nit-pick, but let's avoid using one-liners and go with egyptian braces instead.